pkgs/tools/misc/zf/default.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  zig,
+  testers,
+  zf,
+  ...
+}:
+stdenv.mkDerivation rec {
+  pname = "zf";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "natecraddock";
+    repo = pname;
+    rev = version;
```

```suggestion
    rev = "refs/tags/${version}";
```

Ensure we're using a tag.

---

done

---

we don't have to do this any more

https://github.blog/changelog/2023-03-23-block-ambiguous-branch-and-tag-names

---

I thought this prevents branches from being used instead of tags.

---

That's not what that blog post says. It says that now "GitHub blocks branch and tag names which begin with refs/."

It doesn't affect existing repositories, so why would we stop being explicit?

Here's a repo I just created that has both tags and branches of the same name, but now on different commits:

https://github.com/adamcstephens/testrepo/releases/tag/v0.0.1
https://github.com/adamcstephens/testrepo/tree/v0.0.1

---

you're right, I misunderstood

---

No problem. I built the test repo just to verify I was understanding it myself. :)